README.md

```diff
@@ -199,7 +204,8 @@ You can also install NumPyro from source:
 ```
 git clone https://github.com/pyro-ppl/numpyro.git
 # install jax/jaxlib first for CUDA support
-pip install -e .[dev]
+pip install -e .
+# or `pip install -e .[funsor]
```

Comment:
typo: `

Reply:
I think it will be nice to include test dependencies into `dev` and mention both `funsor` (for end users experimenting with more advanced machinery) as well as `dev` for users who are looking to do numpyro development either as contributors or for tinkering things on their own system.

Reply:
So we will add `funsor` to `dev` and keep `pip install -e .[dev]` here? That looks good to me.

Reply:
Either is fine. Keeping it separate is okay, we should just mention the use cases.

```
pip install -e .[funsor]  # for experimental `numpyro.contrib.funsor` module for inference on models with discrete variables
pip install -e .[dev]  # contains additional dependencies for NumPyro development
```